.gitignore

```diff
@@ -0,0 +1,37 @@
+HELP.md
+target/
+!.mvn/wrapper/maven-wrapper.jar
+!**/src/main/**/target/
+!**/src/test/**/target/
```

Comment:
There are lots of file references here that don't exist. Try and keep the file clean for those things you know you need to exclude